app/src/index.css

```
@@ -271,6 +292,10 @@
   --animate-float: float 3s ease-in-out infinite;
   --animate-ticker: ticker 30s linear infinite;
   --animate-wiggle: wiggle 0.5s ease-in-out;
+  /* The composer's ambient shadow orbit — see the `composerShadowOrbit`
+     keyframes below. `linear`, not an ease: the offset traces a circle, and an
+     ease would make the light appear to speed up and stall twice per lap. */
+  --animate-composer-shadow: composerShadowOrbit 18s linear infinite;
```

Stylelint (17.14.0)

[error] Expected "composerShadowOrbit" to be "composershadoworbit" (value-keyword-case)

(value-keyword-case)